tests/CancellableEventLoopTest.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace M6WebTest\Tornado;
+
+abstract class CancellableEventLoopTest extends EventLoopTest
```

Comment:
What is the purpose of this new class? It doesn't define any abstract function… so why not include the trait and the constant in the base `EventLoopTest`? 🤔